src/pages/Search/SearchPageBottomTab.tsx

```diff
@@ -27,17 +27,19 @@ function SearchPageBottomTab() {
     const {clearSelectedTransactions} = useSearchContext();
     const [selectionMode] = useOnyx(ONYXKEYS.MOBILE_SELECTION_MODE);
 
-    const {queryJSON, policyID} = useMemo(() => {
+    const EXPENSE_FILTERS_TYPE = 'expense';
```

Comment:
We already have this in `CONST.SEARCH.DATA_TYPES.EXPENSE`, but I don't think we need this. See my comments below

Reply:
Answered you [here](https://github.com/Expensify/App/pull/48566#discussion_r1759412682)

Reply:
If we use `SearchUtils.isCannedSearchQuery` we don't need this anymore, do we?

Reply:
Yes we don't need this anymore